WalletWasabi/Bases/ConfigManager.cs

```diff
@@ -41,6 +50,41 @@ private static TResponse LoadFile<TResponse>(string filePath)
 
 		return result is not null
 			? result
-			: throw new("Unexpected null value.");
+			: throw new JsonException("Unexpected null value.");
+	}
+
+	public static TResponse LoadFile<TResponse>(string filePath, bool createIfMissing = false)
+		where TResponse : IConfigNg, new()
+	{
+		TResponse result;
+
+		if (!createIfMissing)
+		{
+			return LoadFile<TResponse>(filePath);
```

Comment:
The same call is wrapped inside a try-catch block below. Shouldn't we do the same here?